library/core/tests/slice.rs

```diff
@@ -224,6 +226,12 @@ fn test_iterator_count() {
     assert_eq!(iter2.count(), 3);
 }
 
+#[test]
+fn test_iterator_default() {
+    let iter: Iter<'_, u8> = Iter::default();
+    assert_eq!(iter.len(), 0);
```

Comment:
For example, this could be an example in the doc comment, for users to see, instead of a normal test.

Reply:
I tried but since it's generated in a macro I get this

```
4 | /// let iter: &mut[] = Default::default();
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ rustdoc does not generate documentation for macro invocations
```

Is there a way around that?

Reply:
Nevermind, figured it out.